### DISCUSSION TITLE
is there a way to set parameter parallel_tool_calls to false for assistant

### DISCUSSION THREAD

**Discussion by ElisMia** [RageAgainstThePixel/com.openai.unity#266] [Q&A] [answered]
Hey, all!
I wanted to ask. is there a way to set parameter parallel_tool_calls to false for assistant? it seems like it's true by default and assistant makes a lot of calls in parallel during the run.

this one https://platform.openai.com/docs/guides/function-calling

**Comment by StephenHodgson** [comment, marked as answer]
add `parallelToolCalls: true` in your `CreateRunRequest`, `CreateThreadAndRunRequest` or `ChatRequest` .ctor

https://rageagainstthepixel.github.io/OpenAI-DotNet/api/OpenAI.Chat.ChatRequest.html#parameters
https://rageagainstthepixel.github.io/OpenAI-DotNet/api/OpenAI.Threads.CreateRunRequest.html#parameters-1
https://rageagainstthepixel.github.io/OpenAI-DotNet/api/OpenAI.Threads.CreateThreadAndRunRequest.html#parameters-1

**Reply by ElisMia** [reply]
thak you a lot!
